lib/ansible/modules/identity/cyberark/cyberark_user.py

```diff
@@ -187,7 +198,6 @@ def user_details(module):
             msg=("Unknown error while performing user_details."
                  "\n*** end_point=%s%s\n%s" % (api_base_url, end_point, to_text(unknown_exception))),
             headers=headers,
-            exception=traceback.format_exc(),
```

Comment:
Any specific reason to remove traceback ? Just curious.

Reply:
We do this automatically in Ansible for fail_json() in except-blocks. No need to do it individually.